docs/resources/alert.md

```diff
@@ -74,9 +74,9 @@ EOT
 
 Optional:
 
-- `exclude_filters` (List of Map of String)
+- `exclude_filters` (List of Map of String) ID of the corresponding destination resource
 - `filters` (List of Map of String) Non-equality filters (operand: contains, regexp, etc)
-- `include_filters` (List of Map of String)
+- `include_filters` (List of Map of String) For alert queries that produce multiple group_by values, if at least one entry is specified for this field, the destination will only receive notification for group_by results that include the set of attributes specified here.
```

Comment:
typo? I don't think `exclude_filters` is an ID

Reply:
while we're here should we remove `exclude_filters` and `filters`? we only support equality

Reply:
heh, yeah, this was a mangled diff from trying to remove `exclude_filters` and `filters` and then thinking better of it and removing those changes -- thanks for noticing! and yeah, I'll remove the filters in a separate PR to avoid holding up this one.